src/main/java/edu/cornell/kfs/sys/CUKFSPropertyConstants.java

```diff
@@ -93,4 +93,10 @@ public class CUKFSPropertyConstants {
 
     public static final String DOCUMENT_ID = "documentId";
     public static final String FINALIZED_DATE = "finalizedDate";
+
+    public static class ISOFIPSCountryMap {
+        public static final String ISO_COUNTRY_CODE = "isoCountryCode";
+        public static final String FIPS_COUNTRY_CODE = "fipsCountryCode";
+        public static final String ACTIVE = "active";
```

Comment:
If you wanted to simplify the setup of the property constants, you could replace this custom "active" property reference with the use of the KFSPropertyConstants.ACTIVE constant.

Reply:
adjusting